ui/admin/app/styles/app.scss

```diff
@@ -735,6 +735,19 @@
   /* stylelint-enable selector-class-pattern */
 }
 
+.config-tags {
+  /* stylelint-disable selector-class-pattern */
+  .hds-table__tbody {
+    .hds-table__td {
+      &:last-child {
```

Comment:
This could have just been `.hds-table__td:last-child`

Reply:
@ZedLi yes, thank you for the catch